tests/time_tests/test_runner/utils.py

```diff
@@ -53,7 +53,7 @@ def metadata_from_manifest(manifest: Path):
         'commit_sha': repo_trigger['revision'],
         'commit_date': repo_trigger['commit_time'],
         'repo_url': repo_trigger['url'],
-        'target_branch': repo_trigger['target_branch'],
+        'branch': repo_trigger['branch'],
```

Comment:
revert renaming because it's already used, and change will break backward compatibility

Reply:
Done